ddtrace/appsec/processor.py

```diff
@@ -34,6 +34,12 @@
 from ddtrace.internal.rate_limiter import RateLimiter
 
 
+try:
+    from json.decoder import JSONDecodeError
+except ImportError:
+    # handling python 2.X import error
+    JSONDecodeError = ValueError  # type: ignore
```

Comment:
We have this logic in a few places. We should define this type in a common module.